plugins-root/check_icmp.c

```diff
@@ -1530,6 +1832,47 @@ get_threshold(char *str, threshold *th)
 	return 0;
 }
 
+/* not too good at checking errors, but it'll do (main() should barfe on -1) */
+static int
+get_threshold2(char *str, threshold *warn, threshold *crit, int type)
+{
+	char *p = NULL;
+	bool i = false;
```

Comment:
Sorry to be annoying, but what "does" `i`? and could it be renamed in a way to not be confused with the typical iterator?

Reply:
I don't know :shrug: I have not written that code. I just tried to make #1503 mergable and fixed some tests. NP is having it and people are requesting it for MP as well.

Reply:
fair enough :-)